From 6dc6b68ab4c95193393f67a777e529eac3bc20ef Mon Sep 17 00:00:00 2001
From: "jonghen.han" <jonghen.han@lge.com>
Date: Thu, 24 Sep 2020 16:16:09 +0900
Subject: [PATCH] webOS: Fix UI bug in iotivity sampler

:Release Notes:
Fix UI bug in iotivity sampler

:Detailed Notes:
This commit fixes the send button is not displayed.

:Testing Performed:
Verified that local build version.

:Issues Addressed:
[PLAT-120446] [VirtualBox_Auto_Emulator][LINUX 18.04][Build#248]
              "SEND" button not displayed properly and "Scroll Bar"
              not moving to press "Send " button.
---
 .../webos/com.example.app.iotivity/src/views/Client.js       | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/resource/csdk/stack/samples/webos/com.example.app.iotivity/src/views/Client.js b/resource/csdk/stack/samples/webos/com.example.app.iotivity/src/views/Client.js
index e1077210a..382ef649b 100644
--- a/resource/csdk/stack/samples/webos/com.example.app.iotivity/src/views/Client.js
+++ b/resource/csdk/stack/samples/webos/com.example.app.iotivity/src/views/Client.js
@@ -323,13 +323,12 @@ class Client extends React.Component {
                                 <ExpandableInput title="value" onChange={this.onResourceValueChanged} />
                                 <Button style={{ lineHeight: 25 + "px" }} disabled={!this.state.resourceUri || !this.state.resourceKey || !this.state.resourceValue} small onClick={this.onPostResourceTapped}>send</Button>
                             </div> : null}
-                        {detailResourceInfo ?
                             <div>
                                 <MarqueeText marqueeOn="render">return</MarqueeText>
                                 <SpotlightContainerDecorator focusableScrollbar style={{ "height": 400 + "px" }} direction="both" horizontalScrollbar="auto" verticalScrollbar="auto">
-                                    <BodyText>{JSON.stringify(detailResourceInfo, null, 4)}</BodyText>
+                                    <BodyText>{(detailResourceInfo ? JSON.stringify(detailResourceInfo, null, 4) : "")}</BodyText>
                                 </SpotlightContainerDecorator>
-                            </div> : null}
+                            </div>
                     </SpotlightContainerDecorator>
                 </Layout>
             </div>
-- 
2.17.1
